src/DoctrineAuditBundle/translations/messages.de.yaml

```diff
@@ -0,0 +1,24 @@
+audit:
+  header:
+    home: Home
+    audited_entities: geprüfte Unternehmen
+  tableHeaders:
```

Comment:
Using camel case here makes the final token look weird as long as other token parts are all using snake case

Reply:
yep ur right :) done!